satellite/payments/stripecoinpayments/customers.go

```diff
@@ -10,6 +10,8 @@ import (
 )
 
 // CustomersDB is interface for working with stripe customers table.
+//
+// architecture: Database
 type CustomersDB interface {
```

Comment:
For implementations also add `var _ stripecoinpayments.CustomerDB = (*customers)(nil)`. That way the archview can pick up the relation between them.